src/app/components/libs/component-events.ts

```diff
@@ -0,0 +1,29 @@
+import { ComponentMetaEx } from '@/app/meta/meta-all';
+import { GET_COMPONENT } from '@/app/store/modules/components/getters';
+import { MetaEvent } from '@ulaval/modul-components/dist/meta/v2';
+import Component from 'vue-class-component';
+
+import { ModulWebsite } from '../modul-website';
+import WithRender from './component-events.html?style=./component-events.scss';
+
+@WithRender
+@Component
+export class ComponentEvents extends ModulWebsite {
+
+    private get component(): ComponentMetaEx | null {
+        return this.$store.getters[GET_COMPONENT];
+    }
+
+    private get hasEvents(): boolean {
+        return this.component.metaV2.events && this.component.metaV2.events.length != 0;
+    }
+
+    private get events(): MetaEvent[] {
+        return this.component.metaV2.events;
+    }
+
+    private getEventDescriptionKey(event: MetaEvent): string {
+        return this.component.metaKey + 'event_' + event.name;
```

Comment:
encore une fois très gros détail
`${this.component.metaKey}event_${event.name}`